src/v/config/property.h

```diff
@@ -41,6 +41,7 @@ class property : public base_property {
 public:
     using validator =
       typename ss::noncopyable_function<std::optional<ss::sstring>(const T&)>;
+    using value_t = T;
```

Comment:
I think this typedef is unused

Reply:
Nice catch! It was there for a change that has been removed from the PR since then.